benchmark/child_process/child-process-read.js

```diff
@@ -19,7 +19,7 @@ function main(conf) {
   const dur = +conf.dur;
   const len = +conf.len;
 
-  const msg = '"' + Array(len).join('.') + '"';
+  const msg = `"${'.'.repeat(len)}"`;
```

Comment:
I'm not sure -- since you only ran the linter, I had assumed that this wasn't intended to make any functional changes to the code. (Since it seems like the functional changes are intentional, please ignore this review.)